components/table/index.en-US.md

```diff
@@ -160,6 +160,18 @@ One of the Table `columns` prop for describing the table's columns, Column has t
 | -------- | ------------------------- | --------- | ------- |
 | title    | Title of the column group | ReactNode | -       |
 
+### loading
+
+Properties for loading.
+
+| Property | Description | Type | Default |
+| --- | --- | --- | --- |
+| type | Loading component to use. Options: `spin` and `skeleton` | string | `spin` |
+
+For more on props to configure `spin` please check [`Spin`](/components/spin/).
+
+For more on props to configure `skeleton` please check [`Skeleton`](/components/skeleton/).
```

Comment:
components/table/index.zh-CN.md

Reply:
@afc163 is there someone that can help me with the translation or do I just google translate?